frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -83,13 +83,13 @@ local symbol_prefix = {
         pages_left = BD.mirroredUILayout() and "<" or ">",
         battery = "",
         -- @translators This is the footer compact letter prefix for the number of bookmarks (bookmark count).
-        bookmark_count = "B",
+        bookmark_count = C_("FooterLetterPrefix", "BM"),
```

Comment:
I think it's better to use another "context name" than `FooterLetterPrefix `, to not confuse the translators.
`FooterCompactLetterPrefix`